data/benefits/javascript/intercommunalite_golfe_du_morbihan_vannes_agglomeration-ticket-sport-culture-et-nature.yml

```
@@ -0,0 +1,36 @@
+label: Ticket Sport Culture et Nature
+institution: ville-de-vannes
+description: La Ville de Vannes organise l’opération « Ticket Sport Culture
+  Nature » pendant les vacances scolaires. Il s'agit de stages culturels ou
+  sportifs proposés à la journée, à la demie-journée ou à la semaine et encadrés
+  par des Éducateurs Municipaux. Attention, les périodes d'inscription étant assez courtes, veillez à bien consulter les dates d'inscription sur le
+  site de la ville.
+prefix: le
+conditions_generales:
+  - type: age
+    operator: ">="
+    value: 7
+  - type: age
+    operator: <=
+    value: 16
+  - type: communes
+    values:
+      - "56260"
+  - type: quotient_familial
+    operator: <=
+    period: month
+    value: 2101
+profils: []
+conditions:
+  - Consulter la liste et le calendrier des activités proposées sur le site de
+    la ville.
+  - Vérifier votre tarif sur le simulateur de la ville de Vannes, en fonction de
+    votre quotient familial.
+  - Créer votre compte sur Vannes & vous.
+type: float
+unit: €
+periodicite: ponctuelle
+legend: € à 10 € par 1/2 journée
```

```suggestion
legend: à 10 € par 1/2 journée
```

Car sinon ça affiche : 
2 €  € à 10 € par 1/2 journée

cf: https://deploy-preview-4357--aides-jeunes.netlify.app/aides/intercommunalite_golfe_du_morbihan_vannes_agglomeration-ticket-sport-culture-et-nature